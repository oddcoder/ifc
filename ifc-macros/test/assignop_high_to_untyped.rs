use ifc_macros::ifc_block;

fn main() {
    let mut x = 5;
    ifc_block!{
        let y = 5;
        x += y;
    };
}