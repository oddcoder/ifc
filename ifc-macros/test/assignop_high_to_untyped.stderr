error: Cannot assign high expression to low variable
 --> $DIR/assignop_high_to_untyped.rs:7:9
  |
7 |         x += y;
  |         ^^^^^^
  |
help: Low Variable
 --> $DIR/assignop_high_to_untyped.rs:7:9
  |
7 |         x += y;
  |         ^
help: High Expression
 --> $DIR/assignop_high_to_untyped.rs:7:14
  |
7 |         x += y;
  |              ^
